src/actions/payments.js

```diff
@@ -66,22 +66,18 @@ class ActionsPayments {
     });
   }
 
-  decodePaymentRequest(paymentRequest) {
+  async decodePaymentRequest(paymentRequest) {
     // Check if lightning address
-    return new Promise((resolve, reject) => {
-      paymentRequest = paymentRequest.replace(PREFIX_URI, ''); // Remove URI prefix if it exists
-      this._actionsGrpc
-        .sendCommand('decodePayReq', {
-          pay_req: paymentRequest,
-        })
-        .then(response => {
-          resolve(response.num_satoshis);
-          // resolve(response);
-        })
-        .catch(err => {
-          reject(err);
-        });
-    });
+    paymentRequest = paymentRequest.replace(PREFIX_URI, ''); // Remove URI prefix if it exists
+    try {
+      const request = await this._actionsGrpc.sendCommand('decodePayReq', {
+        pay_req: paymentRequest,
+      });
+      this._store.paymentsResponse = request;
```

Comment:
Love the use of async/await 👍For consistency's sake we should map the comand's [response attributes](http://api.lightning.community/#decodepayreq) to camelCase attributes like in the [channel actions](https://github.com/lightninglabs/lightning-app/blob/abeb8839b2dbf382fb17b459609c232c3f206447/src/actions/channels.js#L36-L45). It's a bit more boilerplate code but readers of the code won't have to look what the return values are in the lnd api docs.